mangaki/mangaki/algo/fma.py

```diff
@@ -44,13 +49,13 @@ def fit(self, X, y):
         X_fm = self.prepare_fm(X)
         self.chrono.save('prepare data in sparse FM format')
 
-        os.environ['LIBFM_PATH'] = 'XXX'  # If applicable
+        os.environ['LIBFM_PATH'] = '/Users/jilljenn/code/libfm/bin/'  # If applicable
```

Comment:
This is developer-specific, let's just document this somewhere and just put `LIBFM_PATH=/Users/jilljenn/code/libfm/bin ./manage.py compare` or whatever, whenever you run this, wouldn't this be better?